tests/javascript/index.php

```diff
@@ -2099,7 +2102,7 @@ function assertTrackingRequest(actual, expectedStartsWith, message)
     });
 
     test("API methods", function() {
-        expect(123);
+        expect(129);
```

Comment:
We removed 4 methods so this needs to go down to
```suggestion
        expect(125);
```